README.md

```diff
@@ -128,11 +128,30 @@ and test your migrations thoroughly.
 
 ## Installation
 
-For now, the project only supports `Cocoapods`. 
+#### CocoaPods
+
 Add the pod to your testing target in `Podfile`:
 
 ```ruby
 target 'MyAppTests' do
   pod 'GRDBSnapshotTesting', '~> 0.1'
 end
-```
\ No newline at end of file
+```
+
+#### Swift Package Manager
+
+Add the package to your testing target in `Package.swift`:
+
+```diff
```

Comment:
Maybe we should use `swift` here instead of `diff`?

Reply:
It was a matter of highlighting the lines added to the Package, when rendered by GitHub. But I can switch to Swift syntax highlighting if you prefer.

Reply:
Only now I noticed those `+` signs at the beginning of lines. It makes complete sense now to use `diff` 👍